go.mod

```diff
@@ -52,8 +52,6 @@ require (
 	github.com/jrick/logrotate v1.0.0
 	github.com/kylelemons/godebug v1.1.0 // indirect
 	github.com/marcopeereboom/sbox v1.0.0
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
```

Comment:
Building this PR adds these two packages back into the go.mod file.  Is removing these intentional?

Reply:
Nope they keep getting removed for me.  Not really sure why though.  Hadn't changed anything wrt deps

Reply:
Must be something on my end.